test/Interpreter/llvm_link_time_opt.swift

```diff
@@ -1,8 +1,6 @@
 // UNSUPPORTED: OS=windows-msvc
 // static library is not well supported yet on Windows
 
-// XFAIL: OS=linux-android, OS=linux-androideabi
```

Comment:
Similarly, [I disabled this last year because it didn't work with clang 14 in NDK 25](https://github.com/swiftlang/swift/pull/65968/files#r1273589720), but it works with clang 17 in NDK 26 now.